app/services/dunning_campaigns/update_service.rb

```diff
@@ -45,21 +32,84 @@ def call
 
     attr_reader :dunning_campaign, :organization, :params
 
+    def update_dunning_campaign_attributes
+      dunning_campaign.assign_attributes(permitted_attributes)
+    end
+
+    def permitted_attributes
+      params.slice(:name, :code, :description, :days_between_attempts, :max_attempts)
+    end
+
     def handle_thresholds
```

Comment:
what about extracting this logic into a separate service `DunningCampaignThresholds::CreateOrUpdateBatchService` or something like this?

as it's quite complex, can be a bit easier to test and maintain, no?

Reply:
lets do it in a separate PR, need to unblock this for the next PR...